team.go

```diff
@@ -299,26 +305,26 @@ func (client *Client) CreateTeam(input TeamCreateInput) (*Team, error) {
 	return &m.Payload.Team, FormatErrors(m.Payload.Errors)
 }
 
-func (client *Client) AddMembers(team *TeamId, emails []string) ([]TeamMembership, error) {
+func (client *Client) AddMembers(team *TeamId, memberships ...TeamMembershipUserInput) ([]TeamMembership, error) {
```

Comment:
Should this be renamed to `AddMemberships`?

Reply:
I considered this and... yes sort of. I'm going to rename this method and create a new Deprecated method named `AddMembers`. That should ease the transition for anybody out there that hypothetically relies on this. Also this will necessitate updates in terraform-provider 👌